jenkins-scripts/dsl/ignition.dsl

```diff
@@ -181,11 +205,16 @@ ArrayList prerelease_branches(String ign_software)
 ArrayList all_branches(String ign_software)
 {
   List<String> branches = new ArrayList<String>();
-  supported_branches("${ign_software}").each { major_version ->
+  supported_ign_branches("${ign_software}").each { major_version ->
     if ("${major_version}") {
       branches.add("ign-${ign_software}${major_version}")
     }
   }
+  supported_gz_branches("${gz_software}").each { major_version ->
```

Comment:
The new CI checker doesn't like this line

```
No such property: gz_software for class: ignition
```

Reply:
hopefully fixed by https://github.com/gazebo-tooling/release-tools/pull/787/commits/312c4a20fbb6b979a73f00f1f99924b84e0a9485

I love having CI for these DSL files!!! 😍